src/bot.ts

```diff
@@ -5,7 +5,11 @@ import { DbEmoji } from "./db/dbdataemoji";
 import { DbEvent } from "./db/dbdataevent";
 import { MatrixUser, RemoteUser, Bridge, Entry, Intent } from "matrix-appservice-bridge";
 import { Util } from "./util";
-import { MessageProcessor, MessageProcessorOpts, MessageProcessorMatrixResult } from "./messageprocessor";
+import {
+    DiscordMessageProcessor,
+    DiscordMessageProcessorOpts,
+    DiscordMessageProcessorMatrixResult,
```

Comment:
I wonder if this could have just been named `DiscordMessageProcessorResult` to save some characters.